Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Ride.hs

```diff
@@ -85,6 +88,11 @@ initializeRide merchantId driver booking mbOtpCode enableFrequentLocationUpdates
   rideDetails <- buildRideDetails ride driver vehicle
 
   QRB.updateStatus booking.id DBooking.TRIP_ASSIGNED
+
+  when (booking.isScheduled) do
```

Comment:
Why only for isScheduled booking? It could be other booking as well. Lets move this also inside config.

Reply:
And please call this function at the end of functions.